### PR TITLE
feat: cart-page

### PR DESCRIPTION
**Pull request by ankit9015**
Cart page for Ecommerce app
cart page

### CODE REVIEW

File: cart-page.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.product-bill .card-body{
+    gap: 2rem;
+}
+
+
+
```
Comment (mariyasada) [inline cart-page.css:6]: remove unnecessary space

File: cart-page.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    background-color: rgba(128, 128, 128, 0.568);
+    text-align: left;
+}
+
+.product-card-list .card-horizontal {
+    padding: 2rem;
+
+}
+
+
```
Comment (mariyasada) [inline cart-page.css:19]: here also

**Review by mariyasada — APPROVED** [review]
look at comments

**Review by ozairkhan2810 — APPROVED** [review]
looks good, can merge

**Pull request merged**